vj4/ui/templates/ranking.html

```diff
@@ -0,0 +1,12 @@
+{% extends "layout/basic.html" %}
+{% block content %}
+<div class="row">
+  <div class="medium-12 columns">
+    <div class="section">
+      <div class="section__body no-padding">
+        {% include "partials/ranking.html" %}
```

Comment:
You don't have to create a partial page for this.

Reply:
It's modified from problem_list,I'll fix it later.